lib/brain.js

```
@@ -3155,16 +3155,7 @@ Brain.prototype.updateBillScreen = function updateBillScreen (blockedCustomer) {
 
   emit('billValidatorPending')
 
-  var billUpdate
-  // BACKWARDS_COMPATIBILITY 7.5.0-beta.1
-  const serverVersion = this.trader.serverVersion
-  if (!serverVersion || semver.lt(serverVersion, '7.5.0-beta.1')) {
-    billUpdate = Tx.billUpdateDeprecated(bill)
-  } else {
-    billUpdate = Tx.billUpdate(bill)
-  }
-
-  return this.fastUpdateTx(billUpdate)
```

Yeah, it's time to remove this. But we need to thing of something for this scenario in specific. Maybe throw a error, maybe do it generically on the machine if versions are too far apart. 

@CrypticaScriptura What do you think?

---

Likely time for a version discrepancy screen on the machine that prevents operation entirely if the particular machine's version is sufficiently distant from that of the server. Would be preferable if reserved for major version releases (as this is, v9.0.0).